examples/taskruns/task-multiple-output-image.yaml

```diff
@@ -67,11 +67,8 @@ spec:
   steps:
   - name: build-and-push-1
     image: busybox
-    command:
-    - /bin/sh
-    args:
-    - -ce
-    - |
+    script: |
+      #!/usr/bin/env sh
       set -ex
       cat <<EOF > /builder/home/image-outputs/builtImage1/index.json
```

Comment:
Not related to this patch, but I think we should change this to a variable.
I think `$(outputs.resources.builtImage1.path)/index.json` should work?